firefox-ios/Client/Frontend/Browser/Toolbars/Redux/AddressBarState.swift

```diff
@@ -598,6 +598,9 @@ struct AddressBarState: StateType, Equatable {
         )
     }
 
+    /// This case can occur when scrolling on homepage or in search view
```

Comment:
This seems to the wrong method for this comment. It should be on `handleHideKeyboardAction(state:)` instead. This method is changing the text in the location view when triggered through an interaction with the search view controller (e.g. appending a search term).

Reply:
oh thanks! not sure it ended up there lol